dietpi/dietpi-software

```diff
@@ -1759,6 +1759,14 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_REQUIRES_PHP[$software_id]=1
 		aSOFTWARE_REQUIRES_SQLITE[$software_id]=1
 		aSOFTWARE_REQUIRES_USERINPUT[$software_id]=1
+    #------------------
+    software_id=137
```

Comment:
Generally ID 137 is indeed free now. However, it is always a bid bad to free an ID and re-use it with the same version update. It is reset to "uninstalled" during update to not have a software title "installed" that does not exist anymore but e.g. when during beta phase someone installs unbound, then on every beta/RC update it is reset to "uninstalled".

We solved this in the past by checking sort of fingerprint if the new software title has been installed, but in case of an APT package that could have been done outside of `dietpi-software` while some configuration has not been applied.

So I think cleaned is to to not use a just freed ID without having it at least for one DietPi version unused =>
Please use `182` here.